src/scss/components/grid/_grid.scss

```diff
@@ -9,3 +9,14 @@
   margin: 0 auto;
   align-items: flex-start;
 }
+
+.Grid--gutters {
```

Comment:
Grid--unstuck ? Grid--spaced ?
I mean it is a bit confusing since gutter is visually the space between two Grid-cells